easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,123 @@
+
+name = 'Xvfb'
+version = '21.1.6'
+
+homepage = 'https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml'
+description = """Xvfb is an X server that can run on machines with no display hardware and no physical input devices.
+ It emulates a dumb framebuffer using virtual memory."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkgconf', '1.9.5'),
+    ('Python', '3.11.3'),
+    ('Meson', '1.1.1'),
+    ('Ninja', '1.11.1'),
+    ('libxslt', '1.1.38'),
+    ('gettext', '0.21.1'),
+    ('Bison', '3.8.2'),
+]
+
+dependencies = [
+    ('X11', '20230603'),
+    ('pixman', '0.42.2'),
+    ('libdrm', '2.4.115'),
+    ('Mesa', '23.1.4'),
+    ('nettle', '3.9.1'),
+    ('libunwind', '1.6.2'),
+    ('XZ', '5.4.2'),
+]
+
+default_easyblock = 'ConfigureMake'
+
+local_xvfb_configopts = "--enable-xvfb --disable-xorg --disable-xnest --disable-xwin "
+local_xvfb_configopts += "--disable-dri --disable-dri2 --disable-dri3 --disable-libunwind "
+local_xvfb_configopts += "--with-fontrootdir=%(installdir)s/share/fonts/X11"
+
+# use 'make V=1' to see compiler commands
+local_xvfb_buildopts = "V=1 "
+
+# use static libraries for nettle & libunwind, so avoid errors like "No rule to make target '-lnettle'"
+local_xvfb_buildopts += 'SHA1_LIBS="$EBROOTNETTLE/lib*/libnettle.a" '
+local_xvfb_buildopts += 'LIBUNWIND_LIBS="$EBROOTLIBUNWIND/lib*/libunwind.a $EBROOTXZ/lib*/liblzma.a"'
+
+default_component_specs = {
+    'source_urls': ['https://www.x.org/archive/individual/font/'],
+    'sources': [SOURCE_TAR_GZ],
+    'start_dir': '%(name)s-%(version)s',
+}
+
+local_font_misc_preconfigopts = "export PKG_CONFIG_PATH=%(installdir)s/lib/pkgconfig:$PKG_CONFIG_PATH && "
+local_font_misc_preconfigopts += "export PATH=%(installdir)s/bin:$PATH && "
+
+components = [
+    ('mkfontscale', '1.2.2', {
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['4a5af55e670713024639a7f7d10826d905d86faf574cd77e0f5aef2d00e70168'],
+    }),
+    ('mkfontdir', '1.0.7', {
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['bccc5fb7af1b614eabe4a22766758c87bfc36d66191d08c19d2fa97674b7b5b7'],
+    }),
+    ('bdftopcf', '1.1', {
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['699d1a62012035b1461c7f8e3f05a51c8bd6f28f348983249fb89bbff7309b47'],
+    }),
+    ('font-util', '1.3.3', {
+        'checksums': ['2094dd4a1ca63a61deb101d2dc618682d6e287cdbe09679502223ac445d277dc'],
+    }),
+    ('font-misc-misc', '1.1.2', {
+        'checksums': ['46142c876e176036c61c0c24c0a689079704d5ca5b510d48c025861ee2dbf829'],
+        'preconfigopts': local_font_misc_preconfigopts,
+    }),
+    ('xkbcomp', '1.4.6', {
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['b216a2c8c0eab83f3dc4a3d5ee2bdf7827b30e49c8907035d0f222138eca0987'],
+    }),
+    ('xkeyboard-config', '2.37', {
```

Comment:
There is a 2.39 available (https://www.x.org/archive/individual/data/xkeyboard-config/),